instagram-page-react/src/pages/profilePage/reducers/profilePage.posts.js

```diff
@@ -0,0 +1,25 @@
+import produce from 'immer';
+import {handleActions} from 'redux-actions';
+
+// lodash
+import _get from 'lodash/get';
+
+// constants
+import { EMPTY_ARRAY } from "../../../app.constants";
+
+// actionTypes
+import ACTION_TYPES from "./actionTypes/profilePage.posts";
+
+const INITIAL_STATE = {
+    data: EMPTY_ARRAY,
+}
+
+const handleSetPost = produce((state, action) => {
+    state.data = _get(action, 'payload.posts');
+})
+
+const handlers = {
```

Comment:
Since this is a constant, we should write this in capitals
handlers <> `HANDLERS`